### PR TITLE
Cloud SQL sqlserver private IP address sample

### PR DESCRIPTION
**Pull request by betsy-lichtenberg**
Intending to add this here:

https://cloud.google.com/sql/docs/sqlserver/configure-private-ip

```release-note:none
```

### CODE REVIEW

**Comment by c2thorn** [comment]
Following up with our email thread: the sqlserver/postgres/mysql variations could be combined into one PR. I've assigned myself to all 3. 
@betsy-lichtenberg what do you think about making this one the variation we actually test, and then skipping the test for the other 2?

**Comment by betsy-lichtenberg** [comment]
Added skip test to the other two.

File: mmv1/products/cgc/terraform.yaml [hunk 1]
```diff
@@ -230,6 +230,16 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           deletion_protection: "false"
         ignore_read_extra:
           - "deletion_protection"
+     - !ruby/object:Provider::Terraform::Examples
```
Comment (c2thorn) [inline mmv1/products/cgc/terraform.yaml:233]: I think this block is off by one space which unfortunately matters for .yaml files

File: mmv1/products/cgc/terraform.yaml [hunk 2]
```diff
@@ -230,6 +230,16 @@ overrides: !ruby/object:Overrides::ResourceOverrides
...
+      - !ruby/object:Provider::Terraform::Examples
+        name: "sql_sqlserver_instance_private_ip"
+        primary_resource_id: "instance"
+        primary_resource_type: "google_sql_database_instance"
+        vars:
+          private_network: "private-network"
+          private_ip_address: "private-ip-address"
+          private_ip_sql_instance: "private-ip-sql-instance"
+        ignore_read_extra:
+          - "root_password"
```
Comment (c2thorn) [inline mmv1/products/cgc/terraform.yaml:242]:
```suggestion
          - "root_password"
          - "deletion_protection"
```
also need the deletion_protection field here

**Review by c2thorn — APPROVED** [review]

**Pull request merged**